projects/igniteui-angular/src/lib/grids/grid/grid-filtering-advanced.spec.ts

```diff
@@ -2048,6 +2051,84 @@ describe('IgxGrid - Advanced Filtering #grid - ', () => {
             expect(rows.length).toEqual(1, 'Wrong filtered rows count');
         }));
 
+        it('should handle advanced filtering correctly when grid columns and data are dynamically changed', fakeAsync(() => {
+            const fixture = TestBed.createComponent(IgxGridAdvancedFilteringDynamicColumnsComponent);
+            grid = fixture.componentInstance.grid;
+            fixture.detectChanges();
+
+            grid.height = '800px';
+            fixture.detectChanges();
+            tick(50);
```

Comment:
if this is the desired grid height set it initially .. the grid resizing is not part of the main tested scenario ..